### PR TITLE
[Nice to have in 2.4] Scroll to 'active' content (ie opened content or opened folder)

### PR DESCRIPTION
**Pull request by Skylsmoi**
#2087

## Checkpoints

The goal of this section is to help code reviewer to do required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

- [x] Code is clear enough
- [x] If there are FIXME in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODO, NOTE or HACK in code, date and developer initials are present
- [x] Automated tests have been written (covering feature or non regression), or *at least* an issue has been created for test implementation

### CODE REVIEW

File: frontend/src/container/WorkspaceContent.jsx [hunk 1]
```diff
@@ -110,7 +110,13 @@ class WorkspaceContent extends React.Component {
       else return
     } else wsToLoad = match.params.idws
 
-    this.loadContentList(wsToLoad)
+    this.loadContentList(wsToLoad).then(() => {
```
Comment (Skylsmoi) [inline frontend/src/container/WorkspaceContent.jsx]: don't use promise callback (`.then()`) prefer the usage of async function

**Comment by Skylsmoi** [comment]
I have asked a question in the issue, depending on the answer, some more changes will be required

File: frontend/src/container/WorkspaceContent.jsx (hunk 1 above)
Comment (metzgegu) [reply frontend/src/container/WorkspaceContent.jsx]: Done

**Comment by Skylsmoi** [comment]
:heavy_check_mark: I approve the current implementation
(since I'm the PR's author, I can't review it.)
Waiting for confirmation about the questions in the issue

**Comment by metzgegu** [comment]
:heavy_check_mark: Scroll to ContentItem if one is open

File: frontend/src/container/WorkspaceContent.jsx [hunk 2]
```diff
@@ -110,7 +110,16 @@ class WorkspaceContent extends React.Component {
       else return
     } else wsToLoad = match.params.idws
 
-    this.loadContentList(wsToLoad)
+    await this.loadContentList(wsToLoad)
+
+    let contentToJump = this.getContentIdOpenedInUrl(match.params)
```
Comment (Skylsmoi) [inline frontend/src/container/WorkspaceContent.jsx]: put the code related to the scroll in a specific function and rename variable `contentToJump` to `contentToScrollTo` please

**Pull request merged**